package.json

```diff
@@ -154,7 +154,7 @@
     "@types/react-virtualized-auto-sizer": "^1.0.1",
     "@types/react-window": "^1.8.5",
     "@types/styled-components": "^5.1.25",
-    "@types/uuid": "^8.3.1",
+    "@types/uuid": "^7.0.3",
```

Comment:
What's this change about?

Reply:
There were a couple places we're using uuid in both devtools proper and replay-next but we didn't have a direct dependency on it. When I brought in @replayio/cypress and bumped the other @replayio dependencies, we had some compilation issues because they're using v8 of uuid and we were assuming v7 with our config. So I've added that dependencies in both packages. It means we have two copies locally but only v7 should be included in the bundle (which we already had from react-tooltip and from where I assume we were pulling for our imports)

Reply:
this specifically is aligning the types import to the version the app was actually using (v7.0.3)

Reply:
Yeah. That makes sense. I hadn't noticed the `@types` prefix. 👍🏼